crates/goose-server/src/routes/session.rs

```diff
@@ -199,11 +206,63 @@ async fn delete_session(Path(session_id): Path<String>) -> Result<StatusCode, St
     Ok(StatusCode::OK)
 }
 
+#[utoipa::path(
+    get,
+    path = "/sessions/{session_id}/export",
+    params(
+        ("session_id" = String, Path, description = "Unique identifier for the session")
+    ),
+    responses(
+        (status = 200, description = "Session exported successfully", body = String),
+        (status = 401, description = "Unauthorized - Invalid or missing API key"),
+        (status = 404, description = "Session not found"),
+        (status = 500, description = "Internal server error")
```

Comment:
these just add to the openapi, you don't *need* to enumerate them, but you can if you want to. (on our other endpoints, I don't think they're always telling the truth)